### PR TITLE
Perl6 to raku + fix links

### PR DESCRIPTION
**Pull request by sumanstats**
+ Perl6 to raku
+ fix the link for travis-ci
+ format README.pod
+ change Perl6 references to Raku in docs
+ remove redundant link https://rakudo.perl6.org/downloads/nqp/: in release_guide.pod
+ reflect correct repository of NQP to http://github.com/Raku/nqp/
+ add link to irc channel making it easy to go to the channel

### CODE REVIEW

File: docs/jvminterop-goals.pod [hunk 1]
```diff
@@ -1,10 +1,10 @@
-=encoding utf8
+Raku=encoding utf8
```
Comment (lizmat) [inline docs/jvminterop-goals.pod:1]: This feels wrong?

**Comment by lizmat** [comment]
Other than the one remark I made: 👍

File: docs/jvminterop-goals.pod (hunk 1 above)
Comment (sumanstats) [reply docs/jvminterop-goals.pod]: Fixed the typo now. Thank you for spotting.

**Comment by lizmat** [comment]
Thank you!

**Pull request merged**